libazureinit/Cargo.toml

```diff
@@ -19,10 +19,12 @@ nix = {version = "0.29.0", features = ["fs", "user"]}
 block-utils = "0.11.1"
 tracing = "0.1.40"
 strum = { version = "0.26.3", features = ["derive"] }
+fstab = "0.4.0"
 
 [dev-dependencies]
 tempfile = "3"
 whoami = "1"
+mockall = "0.11.0"
```

Comment:
Now that it does not use `mockall`, can you please remove this line?

Interesting, `cargo machete` does not detect the case.